### PR TITLE
docs: form descriptions and readme

### PR DESCRIPTION
**Pull request by 0xpatrickdev**
- [x] [docs: improve form descriptions](https://github.com/0xpatrickdev/agoric-gov-proposal-hub/commit/bfd454d79d9059d24d2b11cf012a19d59e44ab07)
- [x] [docs: readme disclaimer + netlify link](https://github.com/0xpatrickdev/agoric-gov-proposal-hub/commit/44e61f6c45d7c0385e5effa46c6152fe9a17f148)

### CODE REVIEW

**Comment by 0xpatrickdev** [comment]
Feedback appreciated on language / descriptions here. There are maybe some spots I missed as well cc @dckc

File: src/App.tsx [hunk 1]
```diff
@@ -180,8 +188,23 @@ const App = () => {
...
-                  description="This is a governance proposal that executes code. You will need to provide a JS Bundle, and a JSON Permit file."
                   msgType="coreEvalProposal"
+                  description={
+                    <>
+                      This is a governance proposal that executes code after a
+                      passing vote. The JSON Permit grants{" "}
+                      <Link href="https://docs.agoric.com/guides/coreeval/permissions.html">
+                        capabilities
+                      </Link>{" "}
+                      and the JS Script can install and start a contract. These
```
Comment (0xpatrickdev) [inline src/App.tsx:199]:
```suggestion
and the JS Script can start a contract. These
```
`Instantiate` is probably the correct word. But less words is better? Also leaving out "... or update"

File: README.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+
+
+# ⚖️ Gov Proposal Builder
+
+[Live Demo](https://agoric-gov-proposal-hub.netlify.app)
+
+
+## Disclaimer
+
+The software is provided "as is", without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose and noninfringement. In no event shall the authors or copyright holders be liable for any claim, damages or other liability, whether in an action of contract, tort or otherwise, arising from, out of or in connection with the software or the use or other dealings in the software.
```
Comment (dckc) [inline README.md:11]: I'm curious where that legalese came from. Is the source willing to be attributed?

File: src/App.tsx [hunk 3]
```diff
@@ -180,8 +188,23 @@ const App = () => {
...
                   title="/agoric.swingset.CoreEvalProposal"
-                  description="This is a governance proposal that executes code. You will need to provide a JS Bundle, and a JSON Permit file."
                   msgType="coreEvalProposal"
+                  description={
+                    <>
+                      This is a governance proposal that executes code after a
+                      passing vote. The JSON Permit grants{" "}
+                      <Link href="https://docs.agoric.com/guides/coreeval/permissions.html">
+                        capabilities
+                      </Link>{" "}
```
Comment (dckc) [inline src/App.tsx:198]: We seem to be able to use `<p>` and `<div>`. Why not `<a>`? Why is a special `<Link>` tag needed?
OK, I see `target="_blank" rel="noopener noreferrer"`. We can't let people make up their own minds about which tab to open links in? We're trying to hide where the links come from? (hosting on https will do that for us, no?)

**Review by dckc — APPROVED** [review]
looks OK

Maybe tackle #9 while you're at it?

File: src/App.tsx (hunk 3 above)
Comment (0xpatrickdev) [reply src/App.tsx:198]: This is a good point, I guess it's only appropriate when paired with an external link icon. `noreferrer` is old relic, that used to be security a requirement when target="_blank".

Updating these to <a href="..." />

File: README.md (hunk 2 above)
Comment (0xpatrickdev) [reply README.md:11]: This is from the MIT license. And that is the plan, hopefully.

**Pull request merged**